packages/fluentui/docs/src/examples/components/RadioGroup/Item/RadioGroupItemExample.shorthand.tsx

```diff
@@ -1,15 +1,11 @@
 import * as React from 'react';
 import { RadioGroup } from '@fluentui/react-northstar';
 
-const handleChange = () => {
-  alert('The radio checked value was changed!');
-};
-
 const RadioGroupItemExample = () => (
   <RadioGroup
     items={[
-      { key: '1', label: 'Make your choice', value: '1', checkedChanged: handleChange },
-      { key: '2', label: 'Another option', value: '2', checkedChanged: handleChange },
+      { key: '1', label: 'Make your choice', value: '1' },
```

Comment:
Looks like those could be just renamed see https://github.com/microsoft/fluent-ui-react/commit/d3e6c152549171ec31944bbddfaf0f232f14445f
```suggestion
      { key: '1', label: 'Make your choice', value: '1', onCheckedValueChange: handleChange},
```